### ISSUE TITLE
oznu/docker-unms is no longer maintained

### ISSUE THREAD

**Issue by oznu** [oznu/docker-unms#53]
The oznu/unms Docker image will no longer be receiving updates.

While this image has been reasonably popular, with over 5 million pulls on Docker Hub, keeping up with the updates and changes being made to UNMS by Ubiquiti is beyond my current capacity.

For the Synology and ARM users, I would suggest raising a feature request on the UNMS forums for offical support to be added: https://community.ui.com/tags/unms/questions.

UNMS version 0.13.3 will still continue to be available via this image.

**Comment by conradKE** [comment]
whaaat

**Comment by JacoboSanchez** [comment]
Well i just want to say thanks for your work

**Comment by Trunkzeh** [comment]
Likewise! Thanks for all the hard work :-)

**Comment by ng7m** [comment]
thanks for your efforts, I just ordered a gigabyte mini pc for a dedicated unms setup.... i suspect keeping up would be a crazy amount of work. ;(

**Comment by CsHeng** [comment]
Thank you anyway.

**Comment by davidm870** [comment]
Thanks for your work!

**Comment by pauloalima** [comment]
Thank you for all the work.

**Comment by skjnldsv** [comment]
Thank you for the amazing work!
Sad to see you go :)

**Comment by skjnldsv** [comment]
@Nico640 seems to have created a new proper fork with a few updates already https://github.com/Nico640/docker-unms

**Comment by Artiik373** [comment]
> @Nico640 seems to have created a new proper fork with a few updates already https://github.com/Nico640/docker-unms

Yeah. I tested it on Ubuntu and it works very well so far. Looking forward to him as new maintainer.
